source/administration/config-settings.rst

```diff
@@ -2345,6 +2359,18 @@ Id Attribute
 | This feature's ``config.json`` setting is ``"IdAttribute": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Attribute
+^^^^^^^^^^^^^^^^^
+(Optional) The attribute in the SAML Assertion used to apply a guest role to users in Mattermost.
```

Comment:
```suggestion
(Optional) The attribute and value in the SAML Assertion used to apply a guest role to users in Mattermost (i.e., "IsGuest=true" or "UserType=Guest").
```
Are there any other attribute/value combinations - wondering about the use of i.e., and whether e.g., might be more appropriate.

Reply:
No I think that is only one.